INSTALL.md

```diff
@@ -162,7 +175,7 @@ Set your `QIITA_CONFIG_FP` environment variable to point to that file (into `.ba
 
 Update paths in the newly copied configuration file to match your settings, e.g. replace /home/travis/ with your user home directory.
 
-Next, make a test environment:
+If you are working on Linux WSL, you will need to start the redis server in a separate Ubuntu window with the command provided [here](https://github.com/qiita-spots/qiita/blob/master/INSTALL.md#start-qiita) before making a test environment. Next, make a test environment:
```

Comment:
@aliu104 Most people won't know what Linux WSL is. I would suggest expanding to Windows Subsystem for Linux (WSL), and then using WSL for any future mentions.